packages/@aws-cdk/aws-glue/README.md

```diff
@@ -173,6 +190,12 @@ new glue.Database(this, 'MyDatabase', {
   databaseName: 'my_database',
 });
 ```
+The Glue Catalog is populated with data using Glue Crawlers. For more information on crawlers, see [CfnCrawler](https://docs.aws.amazon.com/cdk/api/v1/docs/@aws-cdk_aws-glue.CfnCrawler.html) construct.
```

Comment:
Link to v1, should be https://docs.aws.amazon.com/cdk/api/v2/docs/aws-cdk-lib.aws_glue.CfnCrawler.html